src/_includes/common/configuration-basics.md

```diff
@@ -0,0 +1,182 @@
+`environment`
+
+Sets the environment. This string is freeform and not set by default. A release can be associated with more than one environment to separate them in the Sentry web UI (for example, `staging` as compared to `prod` or similar).
+
+By default the SDK will try to read this value from the `window` environment variable.
+{% endif %}
+{% if include.hide_error_types == nil %}
+`error_types`
+
+Sets which errors are reported. It takes the same values as PHP’s [error_reporting](https://www.php.net/manual/en/errorfunc.configuration.php#ini.error-reporting) configuration parameter.
+
+By default all types of errors are be reported (equivalent to `E_ALL`).
+{% endif %}
+{% if include.hide_error_sample_rate == nil %}
+`sampleRate`
+
+Configures the sample rate as a percentage of events to be sent in the range of `0.0` to `1.0`. The default is `1.0` which means that 100% of events are sent. If set to `0.1` only 10% of events will be sent. Events are picked randomly.
+{% endif %}
+{% if include.hide_max_breadcrumbs== nil %}
+`maxBreadcrumbs`
+
+This variable controls the total amount of breadcrumbs that should be captured. This variable defaults to `100`.
+{% endif %}
+{% if include.hide_attach_stacktrace == nil %}
+`attachStacktrace`
+
+When enabled, stack traces are automatically attached to all messages logged. Stack traces are always attached to exceptions; however, when this option is set, stack traces are also sent with messages. This option, for instance, means that stack traces appear next to all log messages. This feature is `off` by default.
+
+Grouping in Sentry is different for events with stack traces and without. As a result, you will get new groups as you enable or disable this flag for certain events.
+{% endif %}
+{% if include.hide_send_default_pii == nil %}
+`sendDefaultPii`
+
+If this flag is enabled, certain personally identifiable information is added by active integrations. By default no such data is sent.
+
+If possible, we recommend that you turn this feature on to send all such data by default, and manually remove what you don’t want to send using our features for managing [Sensitive Data](/data-management/sensitive-data/).
+{% endif %}
+{% if include.hide_server_name == nil %}
+`serverName`
+
+Can be used to supply a “server name.” When provided, the name of the server is sent along and persisted in the event. For many integrations the server name actually corresponds to the device hostname even in situations where the machine is not actually a server. Most SDKs will attempt to auto-discover this value.
+{% endif %}
+{% if include.hide_allow_urls == nil %}
+`blocklistUrls`
+
+A list of strings or regex patterns that match error URLs that should not be sent to Sentry. This is a "contains" match to the entire file URL. As a result, if you add `foo.com` to it, it will also match on `https://bar.com/myfile/foo.com`. By default, all errors will be sent.
+{% endif %}
+{% if include.hide_deny_urls == nil %}
+`whitelistUrls`
```

Comment:
```suggestion
`denyUrls`
```

Reply:
ref: https://github.com/getsentry/sentry-docs/pull/1825 fixing it in existing docs

Reply:
Isn't whitelist=allow and blacklist=deny?